src/myr/reference.js

```diff
@@ -363,6 +363,26 @@ let transformations = [
         description: <span>The setPhiLength function sets the phi length of the cursor in degrees. This changes the number of degrees shown of certain shapes. Interesting shapes can be achieved with values greater than 360. The default phi length is 360.</span>,
         example: "setPhiLength"
     },
+    {
```

Comment:
Are this actually transformations?

Reply:
Its a bit of a stretch I know, but they're more associated with transformations than geometry, animations, or groups. As random and randomInt would likely be used mostly in conjuntion with transformations like setPosition, setScale, and setColor (plus getRandomColors was already in transformations), it seemed like the most natural section of the four if we dont want to make a 5th category.